### PR TITLE
Improved french translation

### PR DESCRIPTION
**Pull request by mutantbroom**
Full Disclosure: This PR is a part of a project for a college course on open-source projects.

I touched up some of the lines and verbiage of the French translation as well as adding several lines that were marked "unfinished". I took several years of French in school but it has been a while so I do not admit to being 100% perfect on these, corrections are welcome

### CODE REVIEW

File: translations/pencil_fr.ts [hunk 1]
```diff
@@ -47,12 +47,12 @@
     <message>
         <location filename="../app/src/actioncommands.cpp" line="133"/>
         <source>No sound layer exists as a destination for your import. Create a new sound layer?</source>
-        <translation>Votre piste son ne peut pas être importée car aucune piste de son n&apos;existe. Créer une nouvelle piste de son ?</translation>
+        <translation>Votre couche sonore ne peut pas être importée car aucune couche sonore n&apos;existe. Créer une nouvelle couche sonore ?</translation>
```
Comment (scribblemaniac) [inline translations/pencil_fr.ts:50]: This warning only comes up when trying to import a individual sound and no sound layer exists to add that sound to. Describing what's being imported as a sound layer is confusing and probably wrong.

File: translations/pencil_fr.ts [hunk 2]
```diff
@@ -131,12 +131,12 @@
         <location filename="../app/src/actioncommands.cpp" line="655"/>
         <source>Remove selected frames</source>
         <comment>Windows title of remove selected frames pop-up.</comment>
-        <translation type="unfinished"/>
+        <translation>Enlever les trames sélectionnées</translation>
```
Comment (scribblemaniac) [inline translations/pencil_fr.ts:134]: Images is used for frames everywhere else, that's probably what you want to use here and immediately below.

**Review by scribblemaniac — CHANGES_REQUESTED** [review]
Thank you for your contribution. However, contributions to our translations should be made in our Transifex project which can be found here: https://app.transifex.com/pencil2d/pencil2d/translations-pencil-ts--master/. Translations from there are then synced to this repository occasionally and before each release.

I am not a native French speaker either, but I have made a couple of notes. If you could take these into consideration, and apply your changes on Transifex, that would be greatly appreciated. If you have any trouble getting access just let us know.

**Comment by Lawrence-Kwok** [comment]
@J5lx the course (assuming this is the same course and uni I took this past quarter) focuses on open source contribution and the teacher gives us a list of projects that students had successfully contributed to in the past, one of which is Pencil2D.

**Pull request closed without merging**